alf/optimizers/optimizers.py

```diff
@@ -45,6 +109,26 @@ def __init__(self,
             clip_by_global_norm (bool): If True, use `tensor_utils.clip_by_global_norm`
                 to clip gradient. If False, use `tensor_utils.clip_by_norms` for
                 each grad.
+            parvi (string): if not ``None``, paramters with attribute
```

Comment:
Can also update README.md to include these two algorithms.

Reply:
Updated.